updates/1/updates.sql

```diff
@@ -1 +1,36 @@
-FILL ME IN
+/*Add token Gameswap*/
```

Comment:
No you can't have a comment in its own line. As you can see from the code that parses this it takes the lines two by two. Comments would mess with this since you now have 3 lines for an entry (if they have a comment).

A comment is not needed everywhere. Only for things that may not be clear like populating a swapped for field.